### PR TITLE
add: Extension for Zig languge support

### PR DESCRIPTION
**Pull request by simson-sillage**
Adds extensions for Zig language support. License is MIT. Is this fine?

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -1420,6 +1420,12 @@
       "id": "teabyii.ayu",
       "repository": "https://github.com/ayu-theme/vscode-ayu"
     },
+    {
+      "id": "tiehuis.zig",
+      "repository": "https://github.com/ziglang/vscode-zig",
+      "version": "0.2.5",
+      "checkout": "master"
```
Comment (jankeromnes) [inline extensions.json:1427]: This repository has no GitHub Releases or dedicated release tag, so there is no need to try and pin or checkout a specific version.

The best we can do here is publish straight from the default branch.

```suggestion
      "repository": "https://github.com/ziglang/vscode-zig"
```

**Review by jankeromnes — APPROVED** [review]
That's perfect, many thanks @simson-sillage!

One minor adjustment to make, but otherwise this is good to go. 👍

**Comment by jankeromnes** [comment]
Update: Auto-publishing worked! 🎉 https://open-vsx.org/extension/tiehuis/zig

@simson-sillage Please kindly confirm that the extension is working as intended.

**Comment by simson-sillage** [comment]
@jankeromnes Yes, it is working as intended. Thank you very much.

**Pull request merged**